### ISSUE TITLE
-NoNewWindow argument in RunCommand did not work with Chrome app

### ISSUE THREAD

**Issue by maexwellius** [mdgrs-mei/outlook-taskbar-notifier#5]
`Start-Process $args[0] -ArgumentList $args[1..($args.Length-1)] -NoNewWindow`
I removed -NoNewWindow argument and it started to work with chrome open or closed.
My Chrome is installed as an app not as software if it makes a difference.

```
            clickActions = @(
                ,@("RunCommand", "chrome.exe", "https://example.com")
```

**Comment by mdgrs-mei** [comment]
Thank you for the report, and I'm really happy to see that there is a user!

I should have added a mode to run commands without `-NoNewWindow` parameter anyway, so I added a new `RunCommandWithNewWindow` action (included in v0.3.0 release). Hope this solves the issue.

https://github.com/mdgrs-mei/outlook-taskbar-notifier/releases/tag/v0.3.0

**Comment by maexwellius** [comment]
Thank you for quick resolving. This is a great simple tool to have non-intrusive way to now about unread e-mails in other folder than your inbox.
